src/components/Breakpoints.css

```diff
@@ -22,6 +22,10 @@
   border-color: var(--breakpoint-active-color);
 }
 
+.breakpoints-list .breakpoint.is-conditional {
+  border-left-color: var(--theme-graphs-yellow);
+}
```

Comment:
so... this will sound petty, but could you move this rule above the `.breakpoint.paused` rule?

I believe if we do that then the yellow border will be replaced with the green paused border.